commitlint.config.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  parserPreset: 'conventional-changelog-conventionalcommits',
+  rules: {
+    'body-leading-blank': [1, 'always'],
+    'body-max-line-length': [2, 'always', 100],
+    'footer-leading-blank': [1, 'always'],
+    'footer-max-line-length': [2, 'always', 100],
+    'header-max-length': [2, 'always', 100],
```

Comment:
Shouldn't max length of header be 50 characters?

Reply:
Typically yes we aim for 50. Depending on who you ask 50 is a guideline. 
It's up to us to decide how we want to move forward

Reply:
I would keep 100, using conventional-commits the messages are much longer

Reply:
sounds good